inst/shiny-server/server/upload.R

```diff
@@ -131,9 +131,13 @@ observeEvent(selected_folders(), {
                      format_detected,
                      'is detected in the uploaded data set... skip the uploaded data'))
 
-  else
-    format <<- format_detected   # set the global data format
-
+  else {
+    if (format_detected == IOHprofiler && format_selected == TWO_COL){
```

Comment:
So for now, TWO_COl format is also detected as the "IOHprofiler" format

Reply:
Yes, at the moment we use the same info-file structure for both the IOHprofiler and the TWO_COL format (since the two-col format was initially just taken as a iohprofiler file with the remaining columns removed).
We can change this to be a separate info-format, but that would require some changes to the reading of info files.